python/balance/controller.py

```diff
@@ -3,17 +3,45 @@
 import datetime
 import json
 import os
+import requests
 from flask import Blueprint
 from google.appengine.api import app_identity
-
 from google.appengine.api import mail
-
 from transaction.model import Transaction
 from user.model import User
 from credit.model import Credit
+from mailjet_rest import Client
 
 balance = Blueprint('balance', __name__)
 
+def send_simple_message(emailList):
+    f=open("mail/template.html", "r")  
```

Comment:
`f = open(...)`

Reply:
I'm thinking if it's worth caching the content of the file (don't think it's relevant though, should be really quick to read and this api seems to be very sparingly used)